proto/qgb/genesis.proto

```diff
@@ -1,7 +1,26 @@
 syntax = "proto3";
 package qgb;
 
+import "gogoproto/gogo.proto";
```

Comment:
Nit, but can you format the alignment for this file, and add a newline at the end?